### DISCUSSION TITLE
idées / ideas

### DISCUSSION THREAD

**Discussion by Fabrice59660** [ogkalu2/comic-translate#28] [Ideas]
Bonjour,
Est il possible d'intégrer une fonction pour voir et modifier la traduction du texte avant sont implémentation.

Idem est il possible d'intégrer une fonction pour modifier la position du texte dans les bulles manuellement 
Merci d'avance pour votre retour 

Hello, Is it possible to integrate a function to see and modify the translation of the text before implementation. 
Same, is it possible to integrate a function to modify the position of the text in the bubbles manually
Thanks a lot

**Comment by ogkalu2** [comment]
@Snake59ex 
Les deux sont possibles mais cela prendrait du temps à mettre en œuvre et honnêtement, je ne travaille pas sur leur mise en œuvre pour le moment.

**Comment by fliker09** [comment]
Can we compromise on a bit different approach then? Like get the cleaned up version (with no in-painted text), an in-painted translated image (for visual reference) and a text file with all the translated text (probably will have to contain info on positioning and styling). The user fixes the text, re-launches the app and ask it to use the clean image and (updated) text file to execute the in-painting. Does it sound more realistic?

**Comment by ogkalu2** [comment]
@Snake59ex @fliker09 
I've been able to implement both of these things now
J'ai réussi à implémenter les deux fonctionnalités maintenant.
